src/main/play-doc/developer/CreatingBundles.md

```diff
@@ -23,13 +23,13 @@ In addition, Docker images can be run from "universal" bundles. The following se
 ## Producing an OCI bundle with Docker
 ConductR is capable of running images that have been exported from Docker. To do this, use the `bndl` tool to convert a docker image into a ConductR bundle. Behind the scenes, we're using the standard OCI Image format so you can be sure your images are free of vendor lock-in. Additionally, this approach doesn't require Docker to be installed in production; ConductR provides everything necessary to run these images.
 
-The following command will load your docker image directly into ConductR.
+The following command will resolve your docker image from the Docker registry and load it directly into ConductR.
```

Comment:
"from the Docker registry" -> "from DockerHub" or "from the DockerHub registry", no?